pkg/standalone/standalone.go

```diff
@@ -55,6 +56,31 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 }
 
+// TODO: replace this with clientcmd
+func GetAPIServerClient(authPath string, apiServerList util.StringList) (*client.Client, error) {
```

Comment:
API is an initialism, so I don't know why it wouldn't be capitalized if HTTP is?

----- Original Message -----
> > @@ -55,6 +56,31 @@ func (h *delegateHandler) ServeHTTP(w
> > http.ResponseWriter, req *http.Request) {
> >  	w.WriteHeader(http.StatusNotFound)
> >  }
> >  
> > +// TODO: replace this with clientcmd
> > +func GetAPIServerClient(authPath string, apiServerList util.StringList)
> > (*client.Client, error) {
> 
> I believe GetAPIServerClient is not the go-zealot-recommended capitalization.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3270/files#r22610358